PS4k/src/com/vitco/core/EngineViewPrototype.java

```diff
@@ -86,9 +86,28 @@ public final int getWidth() {
         return container.getWidth();
     }
     // get the image rendered in container in high quality
-    public BufferedImage getImage() {
+    public BufferedImage getImage() throws Exception {
         return container.getImage();
     }
+
+    // get camera position
+    public final float[] getCamPosition() { return this.camera.getCamPosition(); }
```

Comment:
Since it's a single line of code, i tends to write such methods like that (Another bad Habit) will change.